src/app/components/forms/game/header/header.ts

```diff
@@ -75,7 +75,11 @@ export default class FormGameHeader extends BaseForm<FormModel>
 		);
 
 		if (result) {
-			this.formModel.$clearHeader();
+			const payload = await this.formModel.$clearHeader();
+			// Overwrite the base model's header media item here.
+			// This needs to be done because this form does not resolve (and may never resolve)
+			// after clearning a header. Need to ensure that the base model's header gets cleared.
+			(this.model as any).header_media_item = payload.game.header_media_item;
```

Comment:
I think we should just assign: `this.model?.assign(payload.game)`; That'll effectively refresh the game with the current data, which is what happens anyway during $clearHeader on the formModel.